website/content/docs/agent/config-entries/ingress-gateway.mdx

```diff
@@ -94,52 +96,42 @@ Listeners = [
 ```
 
 </Tab>
-<Tab heading="JSON">
+<Tab heading="Kubernetes YAML">
```

Comment:
@freddygv I removed the JSON tabs and replaced them with Kubernetes YAML because with 6 tabs the last 2 tabs were pushed off the screen and it wasn't clear that you could side-scroll to even see them. I thought it was likely that json wasn't being used too often and also the other config entries only have HCL. What do you think though?

Reply:
I don't think we should get rid of JSON since that's what the HTTP API accepts. It might even be more common than HCL for that reason.

@jescalan Any suggestions for how to handle 6 tabs here? Are nested tabs possible? Like if we had OSS/Enterprise at one level and JSON/HCL/YAML below.

Reply:
Yeah that's true. I've added them back. This is what it looks like: 
![image](https://user-images.githubusercontent.com/1034429/103679816-30c75780-4f3a-11eb-8791-3e9738271976.png)

Reply:
Nested tabs are indeed possible if that's how you want to present it!

Reply:
Is that supported by the component already?

Reply:
It is, and it has been implemented already in several areas of learn.hashicorp.com

Reply:
How do I use the nested tabs?

Reply:
Just nest the components 😁 

```jsx
<Tabs>

<Tab heading='foo'>

<Tabs>
<Tab heading='bar'>

Content

</Tab>
</Tabs>

</Tab>
```

Reply:
🤯